tools/samm-cli/src/test/java/org/eclipse/esmf/ProcessLauncher.java

```diff
@@ -48,9 +47,9 @@ public ExecutionResult runAndExpectSuccess( final String... arguments ) {
       return result;
    }
 
-   public static record ExecutionContext( List<String> arguments, Optional<byte[]> stdin, File workingDirectory ) {
+   public static record ExecutionContext(List<String> arguments, Optional<byte[]> stdin, File workingDirectory) {
    }
 
-   public static record ExecutionResult( int exitStatus, String stdout, String stderr, byte[] stdoutRaw, byte[] stderrRaw ) {
+   public static record ExecutionResult(int exitStatus, String stdout, String stderr, byte[] stdoutRaw, byte[] stderrRaw) {
```

Comment:
Undo these changes; it seems the formatting style is up to date. I'll have a look at https://github.com/eclipse-esmf/esmf-sdk/issues/25 for this.